app/javascript/components/search/results/StudySearchResult.js

```diff
@@ -129,33 +142,57 @@ function facetMatchBadges(study) {
 }
 
 /* displays a brief summary of a study, with a link to the study page */
-export default function Study({ study }) {
-  const termMatches = study.term_matches
-  const studyTitle= highlightText(study.name, termMatches).styledText
-  const studyDescription = formatDescription(study.description, termMatches)
-  const displayStudyTitle = { __html: studyTitle }
-
-  let inferredBadge = <></>
-  if (study.inferred_match) {
-    const helpText = `${termMatches.join(', ')} was not found in study metadata,
+export default function StudySearchResult({ study }) {
+  if (!study.trd_result) {
+    const termMatches = study.term_matches
+    const studyTitle = highlightText(study.name, termMatches).styledText
+    const studyDescription = formatDescription(study.description, termMatches)
+    const displayStudyTitle = { __html: studyTitle }
+
+    let inferredBadge = <></>
+    if (study.inferred_match) {
+      const helpText = `${termMatches.join(', ')} was not found in study metadata,
      only in study title or description`
-    inferredBadge = <span className="badge soft-badge" data-toggle="tooltip" title={helpText}>text match only</span>
-  }
+      inferredBadge = <span className="badge soft-badge" data-toggle="tooltip" title={helpText}>text match only</span>
+    }
 
-  return (
-    <>
-      <div key={study.accession}>
-        <label htmlFor={study.name} id= 'result-title'>
-          <a href={study.study_url} dangerouslySetInnerHTML = {displayStudyTitle}></a>{inferredBadge}
-        </label>
-        <div>
-          <span className='badge badge-secondary cell-count'>
-            {study.cell_count} Cells
-          </span>
-          { facetMatchBadges(study) }
+    return (
+      <>
+        <div key={study.accession}>
+          <label htmlFor={study.name} id="result-title" className={'study-item'}>
+            <a href={study.study_url} dangerouslySetInnerHTML={displayStudyTitle}/>{inferredBadge}
+            <span className="badge badge-secondary study-type"
+              data-toggle="tooltip"
+              title={'Study comes from Single Cell Portal'}>
+              SCP </span>
```

Comment:
That a study in SCP comes from SCP ought to be implied, I think.

By analogy, Google search marks results that are not default -- e.g. with a small "PDF" badge.  Like Google not marking results that are web pages with an "HTML" badge, I think not marking SCP study results with "SCP" would improve visual signal-to-noise ratio.


```suggestion
```

(I see Devon beat me to this suggestion, but thought this inline concurrence might help.)